src/utils/unit_test/protobuf_utils_test.cpp

```diff
@@ -30,7 +30,7 @@
 #include "lbann/utils/protobuf/impl.hpp"
 
 #include <google/protobuf/descriptor.h>
-#include <protobuf_utils_test_messages.pb.h>
+#include "lbann/proto/protobuf_utils_test_messages.pb.h"
```

Comment:
```suggestion
#include "protobuf_utils_test_messages.pb.h"
```
This is private-to-the-testing, so it does not need the same treatment.